source/deployment/mobile-app-deployment.rst

```diff
@@ -94,28 +84,26 @@ At the same time, you should begin thinking about technical and security require
 - Is a VPN connection required?
 - Do we use an Enterprise Mobile Management (EMM) Provider?
 
-Again, this means pausing before you jump into your Mattermost mobile deployment. This takes a little more time, but will make the difference in ensuring your see a return on your investment.
+Again, this means pausing before you jump into your Mattermost mobile deployment. But it will make all the difference in ensuring you see a return on your investment.
 
 ----
 
 ###################################
-Mattermost Mobile Deployment Primer
+Mattermost mobile deployment primer
 ###################################
 
-A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
-
-We’re here to support you.
+While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.
 
 This primer serves as an introduction to Mattermost mobile technologies. It is a complement to our technical documentation, and focuses on three main areas:
 
 - Technical Requirements
 - Basic Features and Technologies
 - Troubleshooting
 
-Regardless of your expertise, we recommend reading this section before starting your deployment.
+Regardless of your expertise, we recommend reading this section before starting your deployment. And keep in mind, we are here to support_ you.
```

Comment:
```suggestion
Regardless of your expertise, we recommend reading this section before starting your deployment. And keep in mind, we are here to support you.
```

Reply:
This is on purpose and should link (at least it works on my local build) to the support section of the document while rendering as `support`.

Reply:
Non-blocking comment: I'm wondering if a link might render better - i.e., as a hyperlink and not the same font that we use for indicating file names (e.g., config.json).

Reply:
Disregard my comment. I've done a local build and the link renders as expected. 

<img width="1030" alt="Screen Shot 2020-06-23 at 12 41 30" src="https://user-images.githubusercontent.com/4223530/85394456-e149ad00-b54e-11ea-97c1-b9569e20bd83.png">